app/views/collections/index.html.erb

```diff
@@ -16,6 +16,7 @@
         <td><%= link_to 'Show', collection %></td>
         <td><%= link_to 'Edit', edit_collection_path(collection) %></td>
         <td><%= link_to 'Destroy', collection, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to t('workspaces.new.heading'), new_collection_workspace_path(collection) %></td>
```

Comment:
I wonder if we should use a different `i18n` string for this.  We may end up changing the heading of that form/page and might not necessarily want that text change to be reflected in the button.